README.md

```diff
@@ -55,6 +55,16 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 
 We're going to be using Trello's API to fetch and save to-do tasks. In order to call their API, you need to first create a Trello account, then generate an API key and token by following the instructions at https://trello.com/app-key.
 
-Once you have both, you can update the .env file to store these 2 values. These values will be called in the app.py file as:
-API_KEY = os.environ.get("api_key")
-TOKEN = os.environ.get("token").
+Once you have both, you can update the .env file to store these 2 values. These values will be called in the app.py file as: API_KEY = os.environ.get("api_key") TOKEN = os.environ.get("token").
+
+## How to run tests
+
+For Selenium tests, Download Firefox beforehand and you will need to download the matching version of the Gecko Driver executable and place it in the root of your project - the selenium driver just uses this under the hood.
+
+If in poetry env:
+For unit and integration tests: run "pytest tests" in the root code directory. 
```

Comment:
Worth noting that you've moved these folders, so you'll now need to run e.g. `pytest todo_app/tests`

Reply:
moved these to outside the todo_app folder so running pytest tests works.